certbot/certbot/plugins/storage.py

```diff
@@ -106,7 +106,7 @@ def put(self, key, value):
         if not self._initialized:
             self._initialize_storage()
 
-        if not self._classkey in self._data.keys():
+        if self._classkey not in self._data.keys():
```

Comment:
I wonder why `pylint` did not complain here. I can fix it in a separate PR?

Reply:
Yes, that is strange. We heavily modified pylint to avoid it to generate a lot of false positives, maybe it also deactivate some of these kind of nice controls. Definitely it worths to try a fix, in a separate PR I agree.